server/handlers/genomic_annotations.go

```diff
@@ -1 +1,155 @@
 package handlers
+
+import (
+	"errors"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/ldsec/medco-connector/restapi/models"
+	"github.com/ldsec/medco-connector/restapi/server/operations/genomic_annotations"
+	utilserver "github.com/ldsec/medco-connector/util/server"
+	"github.com/sirupsen/logrus"
+	"strings"
+)
+
+var genomicAnnotationsTypes = []string{"variant_name", "protein_change", "hugo_gene_symbol"}
```

Comment:
How so? I thought we have only those three possible types of annotations, which are mapped to the three tables in the db, and based on this assumption I implemented the function that builds the queries and the cli. So I cannot assume to have a list of all possible annotations type at compile time? Where do I find this information at run time?

Reply:
The annotations available depends on the dataset. Actually in this demo dataset we are making available just 3 annotations, but there are many more. So indeed you cannot assume a list at compile time.

To retrieve it during the runtime I see two main options:
- Leave the loader as it is, and retrieve the list of tables in the database in order to get the list of annotations.
- Modify the loader to add the list of annotations in a separate table, and use it to retrieve in the runtime.

Note that there is a chance that currently the loader has hard-coded the list of annotations loaded, but if that's the case that will need to change in the near future.

Reply:
Yes, at the moment everything is hardcoded. Changing the loader in this respect seems to be entail a relevant amount of work, I think we should address it in a dedicated pull request.

Reply:
OK good to know. Agreed with modifying loader being a separate work. Is it OK for you to implement the first option (get list of tables to get the different annotations) in this PR?

Reply:
Since the name of tables is hard-coded, I do not think it makes much sense to spend time to have something dynamic to discover them at the moment. Also because I think that in the future we should not rely on table names to build the queries, because it forces us to do that tricky things we discussed to avoid sql injection. I think we should put all the annotations in a table, with a column indicating the type. So I think that the connector and the loader should be modified consistently at the same time, but if you think it is worth it to modify only the connector for now with a temporary fix I'll do it.

Reply:
My intention was mainly so that we can be done with the part of `medco-connector` already now. If you tell me it is not OK to rely on the table names for the future, then we can drop that for the moment and leave a todo for when we change `medco-loader`, that's good with me.